admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/entity/include/entity-type-list.jsp

```diff
@@ -49,49 +50,50 @@
 </s:if>
 <s:else>
     <ol class="breadcrumb page-tabs-header breadcrumb-position">
-        <li class="page-title-container"><s:text
-                name="%{'title.' + entityManagerName + '.management'}"/></li>
+        <li>
+            <s:text name="breadcrumb.app" /></li>
+        <li>
+            <s:text name="breadcrumb.jacms"/>
+        </li>
```

Comment:
we can not put here specific property keys. please, use the pattern `name="%{'_something_.' + entityManagerName + '._somethingelse_'}"`